securedrop_client/export.py

```diff
@@ -237,7 +237,8 @@ def _create_archive(
         archive_path = os.path.join(archive_dir, archive_fn)
 
         with tarfile.open(archive_path, "w:gz") as archive:
-            cls._add_virtual_file_to_archive(archive, cls.METADATA_FN, metadata)
+            if (metadata):
```

Comment:
as you mentioned, parens can go
```suggestion
            if metadata:
```